ui/eslint.config.js

```diff
@@ -8,12 +8,7 @@ const config = defineConfig([
   ...createConfig(opts),
   {
     rules: {
-      'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/control-has-associated-label': 'off',
-      'react/destructuring-assignment': 'off',
-      'react/no-multi-comp': 'off',
-      'react/no-unused-class-component-methods': 'off',
-      'react/prop-types': 'off',
```

Comment:
The only rule I keep disabled. It's just not practical to fix, even on new code. It can be re-enabled once the entire codebase is written in TypeScript... https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/prop-types.md